lab01-util/src/main/java/by/artempvn/lab01/util/StringUtils.java

```diff
@@ -0,0 +1,16 @@
+package by.artempvn.lab01.util;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class StringUtils {
+
+	public static boolean isPositiveNumber(String str) {
+		boolean isPositive = false;
+		if (str != null && NumberUtils.isParsable(str)) {
+			double number = Double.parseDouble(str);
+			isPositive = (number > 0);
+		}
+		return isPositive;
```

Comment:
It is not a good practice to use nullable parameters for Java methods. Method overloading and decorator pattern are more preferable ways to handle such situations.
Taking this in account, implementation could be simplified to something like:

```suggestion
		if (NumberUtils.isParsable(str)) {
			return Double.parseDouble(str) > 0;
		}
		return false;
```

Reply:
Is this good practise to do some action in _return_ statement like:
`return (NumberUtils.isParsable(str) && Double.parseDouble(str) > 0)` ?
Also I've got recomendation not to use multiple _return_ statement.  So in real projects it doesn't matter?

Reply:
Yep, I'd prefer `return NumberUtils.isParsable(str) && Double.parseDouble(str) > 0;`. Regarding having multiple `return` statements - it depends on concrete situation. Sometimes it indicates that method have more than one responsibility. However, it is better than mutating some variable cause you don't have to track it's state.

Reply:
This behavior could be also implemented using Optional API:
````
Optional.of(str)
    .filter(NumberUtils::isParsable)
    .map(Double::parseDouble)
    .filter(value -> value > 0)
    .isPresent(); 
````

However, in this case `return NumberUtils.isParsable(str) && Double.parseDouble(str) > 0;` looks more simple.

Reply:
Static imports could also be used in order to improve readability: `return isParsable(str) && parseDouble(str) > 0;`